docs/server/source/dev-and-test/index.rst

```diff
@@ -6,3 +6,6 @@ Develop & Test BigchainDB Server
 
    setup-run-node
    running-all-tests
+   setup-bdb-host
```

Comment:
Let's move the `running-all-tests` page last and include links to the Vagrant and Ansible pages here too. Those can be added using syntax like:

```text
   setup-bdb-docker
   ../appendices/run-with-vagrant
   ../appendices/run-with-ansible
   running-all-tests
```